types/vfile/vfile-tests.ts

```diff
@@ -28,5 +29,28 @@ console.log(file.messages);
 const custom: string = file.custom; // 'Custom tango'
 const dataCustom: number = file.data.custom; // 12345
 
+// message method accept any Stringifiable Position(e.g. point, position, node with position)
+const startPoint: Point = {
+    line: 1,
+    column: 1,
+};
+const position: Position = {
+    start: startPoint,
+    end: {
+        line: 2,
+        column: 2,
+    }
+};
+// Accept Point
+file.message('test', position);
```

Comment:
Shouldn’t this one be `startPoint`?

Reply:
Yea, you're right. I've already fixed it since https://github.com/DefinitelyTyped/DefinitelyTyped/pull/19583/commits/9a6f7da2bb64a1a6d5bcdb28c808038ac74db214.